src/utils/beneficial.owner.other.ts

```diff
@@ -60,6 +63,13 @@ export const getBeneficialOwnerOtherById = (req: Request, res: Response, next: N
       [StartDateKey]: startDate
     };
 
+    // Redis removal work - Add extra template options if Redis Remove flag is true
+    const isRegistration: boolean = req.path.startsWith(config.LANDING_URL);
+    if (isActiveFeature(config.FEATURE_FLAG_ENABLE_REDIS_REMOVAL) && isRegistration) {
+      templateOptions.FEATURE_FLAG_ENABLE_REDIS_REMOVAL = true;
```

Comment:
Could this be extracted to a method for consistency?  See addCeasedDateToTemplateOptions(). It also uses different syntax to add the filelds.

Reply:
yes can do, i've now extracted it into a new module/file utils/template.data.ts, a function called addActiveSubmissionBasePathToTemplateData.